docs/API.md

```diff
@@ -364,7 +364,17 @@ The following tags are available to override Datadog-specific options:
 
 <h2 id="opentelemetry-api">OpenTelemetry Compatibility</h2>
 
-This library is OpenTelemetry compliant. Use the [OpenTelemetry API](https://opentelemetry.io/docs/instrumentation/js/) and the Datadog Tracer (dd-trace) library to measure execution times for specific pieces of code. In the following example, a Datadog TracerProvider is registered with @opentelemetry/api:
+This library is OpenTelemetry compliant. The OpenTelemetry API packages (`@opentelemetry/api` and `@opentelemetry/api-logs`) are peer dependencies and must be installed separately.
```

Comment:
Could we add a warning in case a user tries to use any of the otel APIs with a missing dependency and a potential fix?

@rochdev or @bengl PTAL. I am uncertain in what way it is fine to log the warning or not.